modules/pagination/client-react/containers/DataProvider.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useCallback } from 'react';
+
+import settings from '../../../../settings';
+
+const generateEdgesArray = quantity => {
+  const arr = [];
+  for (let i = 1; i <= quantity; i++) {
+    arr.push({ cursor: i, node: { id: i, title: 'Item ' + i } });
+  }
+  return arr;
+};
+
+const itemsNumber = settings.pagination.web.itemsNumber;
+const allEdges = generateEdgesArray(47);
+
+export const useDataProvider = () => {
+  const [items, setItems] = useState(null);
+
+  useEffect(() => {
+    loadData(0, 'replace');
```

Comment:
Why data delivery type (`'replace'`) is hardcoded?

Reply:
Fixed